cmd/manager/main.go

```diff
@@ -153,7 +150,7 @@ func main() {
 	}
 
 	// Setup all Controllers
-	if err := controller.AddToManager(mgr, options.ControllerOptions{SyncSender: statusReporter}); err != nil {
```

Comment:
Should the migration logs below reference `OperatorSource` now?

Reply:
Yes they should, missed this one. Updating it